scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1288,6 +1288,23 @@ String getTaskServerLogsByTag(String id, String taskTag)
     Page<JobInfo> getJobs(int offset, int limit, JobFilterCriteria filterCriteria,
             List<SortParameter<JobSortParameter>> sortParameters) throws NotConnectedException, PermissionException;
 
+    /**
+     * Returns a list of jobs info corresponding to the given job IDs (in the same order)
+     *
+     * @param jobsId
+     *             the list of id of the jobs to return
+     *
+     * @return jobs list of the given list of id
```

Comment:
For example, we can say: @return jobs list of the given list of id, in the same order
This is useful information for anybody wishing to use this method.